Formula/ocaml.rb

```diff
@@ -32,7 +32,7 @@ class Ocaml < Formula
 
   pour_bottle? do
     # The ocaml compilers embed prefix information in weird ways that the default
-    # brew detection doesn't find, and so needs to be explicitly blacklisted.
+    # brew detection doesn't find, and so needs to be explicitly blocked.
```

Comment:
I'm unsure why `infer` failed to build on Intel Big Sur. It was built/tested only because of `ocaml`, which has a syntax-only change in a comment.

If we can ignore this failure, the PR should be good to merge, I think.

Reply:
Works for me. Thanks @nandahkrishna!